src/ARPlanes.js

```diff
@@ -85,6 +88,7 @@ class ARPlanes extends Object3D {
     if (planeObj) {
       this.add(planeObj);
       this.planes.set(plane.identifier, planeObj);
+      this.anchorManager.add(planeObj);
```

Comment:
Are you sure we want to do this? These planes are managed by the plane system and should automatically be "anchored". They get updated based on plane update events already.

Reply:
I was not sure if the plane system was updating the planes when the camera tracking changed, I thought they only where being updated when the extent changes. I will roll back the use of the AnchorManager in the ARPlanes as per our conversation and your conviction that the pose of the planes is also updated.